.github/workflows/ci.yaml

```diff
@@ -22,7 +23,10 @@ jobs:
       - run: cp .github/workflows/kernel-${{ matrix.mode }}.config .config
```

Comment:
we also have to make this a condition, `.config` has to life under `./build`

Reply:
Indeed, good catch.